### DISCUSSION TITLE
Add keyboard shortcuts for "next site" and "previous site"

### DISCUSSION THREAD

**Discussion by purplespider** [plausible/analytics#1037] [Anything Else]
To make it easy to cycle through all your sites, it would be handy if there were keyboard shortcuts for "next site" and "previous site".

Can I propose using `J` and `K`, like Gmail?
Or perhaps `<` and `>`?

(I'm aware you can use `1`-`9`, but this is more useful for jumping to a specific site, and doesn't help if you have more than 9 sites)

**Comment by ukutaht** [comment]
Sounds useful. I like `j` and `k` the most I think.
